src/thing/get.test.ts

```diff
@@ -871,6 +885,284 @@ describe("getDatetimeAll", () => {
   });
 });
 
+describe("getTime", () => {
+  it("returns the time value for the given Predicate", () => {
+    const thingWithTime = getMockThingWithLiteralFor(
+      "https://some.vocab/predicate",
+      "13:37:42Z",
+      "time"
+    );
+    const expectedTime = {
+      hour: 13,
+      minute: 37,
+      second: 42,
+    };
+
+    expect(getTime(thingWithTime, "https://some.vocab/predicate")).toEqual(
```

Comment:
This holds for the other tests as well, but if you use `toStrictEqual` it will also fail the test if there are additional properties that we did not expect. (You might not see this everywhere in the codebase yet since it's a lesson we learned after many tests that still use regular `.toEqual` were already written.)

```suggestion
    expect(getTime(thingWithTime, "https://some.vocab/predicate")).toStrictEqual(
```